Makefile

```diff
@@ -7,3 +7,6 @@ race:
 	go install -v -race ./...
 	go test -c -v -race ./inventory
 	go test -c -v -race ./server
+
+run:
+	go test ./...
```

Comment:
Then why we keep building test binaries above? :) Remove it. And add `-race` flag there.